src/server/index.js

```diff
@@ -1,8 +1,18 @@
 const express = require('express');
+const DatabaseHealth = require('./database-check');
 
 const app = express();
 const PORT = process.env.PORT || 3000;
+const okStatus = {status: 'ok', database: 'ok'};
+const errorStatus = {status: 'ok', database: 'error'};
 
-app.listen(PORT, function() {
+app.get('/heartbeat', function (req, res) {
+  DatabaseHealth.checkDatabaseHealth((isWorking) => {
+    isWorking ? res.json(okStatus) :
```

Comment:
maybe it would be better to format it in a different way, not too readable for me